ui/src/components/Common/TitleWithActions.tsx

```diff
@@ -1,27 +1,41 @@
 import React from 'react';
-import { Header } from 'semantic-ui-react';
+import { useHistory } from 'react-router-dom';
+import { Header, Button } from 'semantic-ui-react';
 import { Link } from 'react-router-dom';
 import { AddPlusIcon } from '../../icons/icons';
+import OverflowMenu, { OverflowMenuEntry } from '../../pages/page/OverflowMenu';
 
 type Action = {
   path: string;
   label: string;
 };
 
 type Params = {
-  actions?: Action[];
+  iconActions?: Action[];
+  otherActions?: Action[];
   title: string;
 };
 
-const TitleWithActions: React.FC<Params> = ({ actions, title, children }) => {
+const TitleWithActions: React.FC<Params> = ({ iconActions, otherActions, title, children }) => {
+  const history = useHistory();
   return (
     <div className="title-with-actions">
       <Header as="h2">{title}</Header>
-      {actions?.map(a => (
+      {iconActions?.map(a => (
         <Link className="a2 with-icon" to={a.path}>
           <AddPlusIcon /> {a.label}
         </Link>
       ))}
+      {otherActions?.map(a => (
+        <Link to={a.path}>
```

Comment:
Need a key here, `a.path` should serve well as I assume it would be unique for all the actions in the array